.github/workflows/pythonapp.yml

```diff
@@ -56,10 +56,3 @@ jobs:
         python depscan/cli.py --purl "pkg:npm/@biomejs/biome@1.8.1"
       env:
         PYTHONIOENCODING: utf-8
-    - name: Generate SBOM with cdxgen
```

Comment:
Shall we keep the sbom generation step somewhere?

Reply:
@prabhu I don't think so - there's nothing this tests that isn't tested elsewhere. If we wanted to make sure the bom gets created by _depscan_ calling cdxgen, that would make sense, but that's not what this did and depscan is already accepting a bom as input in other test cases.

Reply:
Understood. The ultimate goal is to also publish the depscan sbom along with the v6 release. Here is an example for vulnerability-db.

https://github.com/AppThreat/vulnerability-db/blob/9ae07558e525fc513977aa4676c7d8dca4897b66/.github/workflows/pythonpublish.yml#L52

Reply:
Ah, gotcha. I will take the VDB example and apply to the pythonpublish.yml workflow for depscan.